kolibri/plugins/coach/assets/src/views/quizzes/CreateExamPage/SidePanels/ResourceSelection.vue

```diff
@@ -848,6 +871,12 @@
       focusFirstEl() {
         this.$refs.textbox.focus();
       },
+      handleGoBack() {
+        this.$router.back();
```

Comment:
I was wondering what does the "go back" button means when we are navigating though the tree. If it means the same behaviour as the browser back button, or if it means go back in the tree. From my perspective I would think that if I click the back button while searching through the tree, it always take me back to the parent folder, but this doesnt happen if we click on the breadcrumbs:


https://github.com/user-attachments/assets/918da58b-91af-47f8-891b-cd7ad2a3f1d7

Reply:
It also happens when we open the section editor:

https://github.com/user-attachments/assets/1db4fd4a-40ce-4f5d-b1a7-3664abe06a88